tests/sentinel/tests/includes/init-tests.tcl

```diff
@@ -41,7 +41,7 @@ test "(init) Sentinels can start monitoring a master" {
         S $id SENTINEL SET mymaster down-after-milliseconds 2000
         S $id SENTINEL SET mymaster failover-timeout 20000
         S $id SENTINEL SET mymaster parallel-syncs 10
-        if {$::leaked_fds_file != ""} {
+        if {$::leaked_fds_file != "" && [exec uname] == "Linux"} {
```

Comment:
i think there's an advantage of running those scripts even if they do nothing.
it increases the test coverage.
so maybe consider putting the `if` inside the script?

Reply:
The problem is we need to rely on the shebang because Sentinel can't accept multiple arguments as a script, and the location of tclsh varies between platforms. We could use `/usr/bin/env` and make sure we at least have that but I think the value is not high.